Doc/library/stdtypes.rst

```diff
@@ -282,7 +282,7 @@ the operations, see :ref:`operator-summary`):
 +---------------------+---------------------------------+---------+--------------------+
 | ``x / y``           | quotient of *x* and *y*         |         |                    |
 +---------------------+---------------------------------+---------+--------------------+
-| ``x // y``          | floored quotient of *x* and     | \(1)    |                    |
+| ``x // y``          | floored quotient of *x* and     | \(1)\(2)|                    |
```

Comment:
Just an unrelated nitpick: I wonder why this table was formatted this way?  Couldn't we use sphinx's footnotes?

Reply:
I suggest opening a separate issue if you want to pursue this; it's out of scope for this PR.